CRM/Uimods/Tools/SearchTableAdjustments.php

```diff
@@ -279,6 +279,14 @@ public static function adjustContributionTable($objectName, &$headers, &$rows, &
       } else {
         $row[UIMODS_STA_PAYMENTINSTRUMENT_FIELD] = '';
       }
+
+      // GP-22895 show revenue recognition date below contribution receive date
```

Comment:
We can optimize performance by loading the revenue recognition date as part of the batch API request in line 181 (`$missing_data = ...`) instead of doing it one by one.